Sources/TuistCore/Graph/GraphTraverser.swift

```diff
@@ -1368,50 +1372,29 @@ public class GraphTraverser: GraphTraversing {
         return target.target.product == .framework
     }
 
-    func isDependencyDynamicTarget(dependency: GraphDependency) -> Bool {
+    private func isEmbeddableDependencyTarget(dependency: GraphDependency) -> Bool {
         switch dependency {
-        case .macro: return false
-        case .xcframework: return false
-        case .framework: return false
-        case .library: return false
-        case .bundle: return false
-        case .packageProduct: return false
+        case .macro, .xcframework, .framework, .library, .bundle, .packageProduct, .sdk:
+            return false
         case let .target(name, path, _):
             guard let target = target(path: path, name: name) else { return false }
-            return target.target.product.isDynamic
-        case .sdk: return false
+            return target.target.product.isDynamic || target.target.product == .staticFramework
```

Comment:
Is `.staticFramework` an embeddable dependency? My understanding is that it needs to embedded only when it has resources.

Reply:
`staticFramework` is embeddable starting from Xcode 15. a cut down framework is embedded. However, what gets embedded is a cut down framework, with all the resources but only a stub Mach-O image.

Reply:
I've added `containsResources` extension to `Target` to embedded `.staticFramework` only when it has resources